### ISSUE TITLE
Bug: Record merging syntax with switch statement adds a semicolon instead of a comma

### ISSUE THREAD

**Issue by ByronBecker** [dfinity/prettier-plugin-motoko#70]
Example case:

```
let person_1 = {
  name = "John";
  ageGroup = #minor";
};

let minor_permissions = { ... };
let adult_permissions = { ... };

let person_1_plus_permissions = person_1 and (
  switch(person1.ageGroup) {
    case (#minor) { minorPermissions };
    case (#adult) { adultPermissions };
  },  // prettier puts a `;` here instead of a `,`, which throws a syntax error
);

```

**Comment by rvanasa** [comment]
Strange! 

It looks like there is an extraneous quotation mark in `ageGroup = #minor";` (at least in the above example). Fixing that causes the code to format as expected for me ([example](https://m7sm4-2iaaa-aaaab-qabra-cai.raw.ic0.app/?tag=2036597547)). 

The most likely underlying cause is that Prettier is falling back to the JavaScript formatter, which can happen if there is a plugin installation issue.

**Comment by di-wu** [comment]
```motoko
let person_1 = {
  name = "John";
  ageGroup = #minor;
};

let minor_permissions = { ... };
let adult_permissions = { ... };

let person_1_plus_permissions = {
  person_1 and (
    switch(person1.ageGroup) {
      case (#minor) { minorPermissions };
      case (#adult) { adultPermissions };
    },
  );  // prettier puts a `;` here instead of a `,`, which throws a syntax error
}
```

**Comment by rvanasa** [comment]
Okay, thanks! I see what you're saying and will look into fixing this.

**Comment by rvanasa** [comment]
Fixed in `v0.2.4`!
